### ISSUE TITLE
Don't playing videos from YouTube.

### ISSUE THREAD

**Issue by kulichin** [ue4plugins/VlcMedia#27]
Don't playing videos from YouTube. How to solve this problem? 
https://www.youtube.com/watch?v=YnKpeMTy4_4&feature=youtu.be

**Comment by gmpreussner** [comment]
Yeah, this is not supported right now, sorry. As you already figured out, the VLC player application uses Lua scripts to resolve the actual content streams in YouTube URLs. This is a VLC player feature, not a LibVLC feature, so we don't have access to that. The plan is to implement the equivalent of the Lua script in C++ in UE4, but I don't have any time to work on that right now.

**Comment by Aszutai** [comment]
Hi gmpreussner, my team are currently working on a unreal project that involves streaming from youtube and other major video websites and we're very interested in getting the play url function working for vlc. Was there any progress on this issue that you can possible update with github? I'm pretty new to programming myself so even if you haven't had time to tackle this issue yet, any hint/help would be much appreciated. Thank you!

**Comment by gmpreussner** [comment]
I haven't had any time to look into this yet, sorry. If you decide to implement the YouTube URL parser yourself, I'd love a pull request. Thanks!

**Comment by jbkempf** [comment]
> This is a VLC player feature, not a LibVLC feature, so we don't have access to that.

This is incorrect. You can use lua inside libVLC, but I agree it's very tricky to get right, because of the location of the scripts.
